packages/cli/src/update-version-notifier.ts

```diff
@@ -0,0 +1,96 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as semver from 'semver';
```

Comment:
Is it make sense to import all module, instead of only needed methods?

Reply:
Nope, Thanks